src/app/zap-templates/zcl/data-model/chip/content-launch-cluster.xml

```diff
@@ -124,4 +124,10 @@ limitations under the License.
     <item name="value" type="CHAR_STRING"/>
     <item name="externalIDList" type="ContentLaunchAdditionalInfo" array="true"/>
   </struct>
+
+  <bitmap name="SupportedStreamingProtocol" type="BITMAP32">
```

Comment:
```suggestion
  <bitmap name="SupportedStreamingProtocosl" type="BITMAP32">
```
is what it's called in the spec, no?

Reply:
I've omitted the plural to have `SupportedStreamingProtocol.ProtocolName` variable. 
The variable is named per spec as an attribute.

Reply:
I can live with it, I guess, but it's going to confuse someone at some point... we should really just try to match the spec names as much as possible within the constraint of "does it compile?"